library/jdbc/forage-jdbc-common/pom.xml

```diff
@@ -27,6 +27,15 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-jta</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-sql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-tx</artifactId>
+            <version>${spring.version}</version>
```

Comment:
Is this OK for Quarkus @JiriOndrusek?

Reply:
yes it is, I already discussed it with @JiriOndrusek  and this needed by Quarkus too